src/scripts/exports.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
+const path = require('path');
+
+const currentDir = process.cwd();
+const args = Array.isArray(process.argv) ? process.argv.slice(2) : [];
+const configFile = (args[0] !== 'clean' && args[0]) || 'package.json';
+const config = JSON.parse(fs.readFileSync(path.join(currentDir, configFile)).toString());
+
+config.exports &&
+	config.exports instanceof Object &&
+	Object.keys(config.exports).forEach(alias =>
+		args[0] !== 'clean'
+			? fs.writeFileSync(
+					path.join(currentDir, alias),
+					`module.exports = require('${path.join(currentDir, config.exports[alias])}');`
```

Comment:
For better user experience, let's check that the target actually exists. Maybe w/ `require.resolve`, so we use the node resolver like will happen at runtime.